src/tableDrawer.ts

```diff
@@ -392,9 +392,12 @@ function drawCellBorders(doc: DocHandler, cell: Cell, cursor: Pos) {
       doc.rect(cell.x, cursor.y, cell.width, cell.height, fillStyle)
     }
   } else if (typeof cellStyles.lineWidth === 'object') {
-    doc.rect(cell.x, cursor.y, cell.width, cell.height, 'F')
 
     const sides = Object.keys(cellStyles.lineWidth)
+    if(sides.length === 4) {
```

Comment:
rect basically creates all 4 borders. I added this condition to keep rendering them if less than 4 sides are specified.